utility.py

```diff
@@ -0,0 +1,139 @@
+from concurrent import futures
+from azure.storage.blob import BlockBlobService, PublicAccess
+import os,uuid,sys
+import io
+import grpc
+import time
+import os.path
+import re
+import config
+import hwsc_file_transaction_svc_pb2
+import hwsc_file_transaction_svc_pb2_grpc
+
+CHUNK_SIZE = 1024 * 1024
+
+def download_chunk(file):
+     with open(file, 'rb')as f:
+         while True:
+             chunk = f.read(CHUNK_SIZE)
+             if len(chunk) == 0:
+                 return
+             yield hwsc_file_transaction_svc_pb2.chunk(buffer=chunk)
+
+def get_file_type(file_name):
+    image_exts_dict = {"jpg": True, "jpeg": True, "png": True, "bmp": True, "tif": True, "gif": True, "tiff": True}
+    audio_exts_dict = {"wav": True, "wma": True, "ogg": True, "m4a": True, "mp3": True}
+    video_exts_dict = {"flv": True, "wmv": True, "mov": True, "avi": True, "mp4": True}
+
+    file_list = file_name.split('.')
+    extension = file_list[-1]
+    file_type = "files"
+
+    if image_exts_dict.get(extension):
+        file_type = "images"
+    elif audio_exts_dict.get(extension):
+        file_type = "audios"
+    elif video_exts_dict.get(extension):
+        file_type = "videos"
+
+    return file_type
+
+def create_uuid_container_in_azure(request):
+    #print(request.uuid)
+    uuid = request.uuid[2:]
+
+    # regular expression that check valid uuid
+    prog = re.compile(r'^[a-zA-Z0-9]{26}$')
```

Comment:
this should be named `uuid_regex` instead of `prog` because it is the `re.compile` returns a regex

Reply:
also `prog` is bad name